web/src/components/roadmap/MiniRoadmap.tsx

```diff
@@ -44,32 +50,65 @@ export const MiniRoadmap = (props: Props): ReactElement => {
     [updateQueue, userData]
   );
 
+  const renderPlanSection = (
+    <SectionAccordion key={sectionNamesInRoadmap[0]} sectionType={sectionNamesInRoadmap[0]} mini={true}>
+      {roadmap?.steps
+        .filter((step) => {
+          return step.section === sectionNamesInRoadmap[0];
+        })
+        .map((step, index, array) => {
+          return (
+            <MiniRoadmapStep
+              step={step}
+              isLast={index === array.length - 1}
+              activeTaskId={props.activeTaskId}
+              completed={isStepCompleted(roadmap, step, userData)}
+              isOpen={userData?.preferences.roadmapOpenSteps.includes(step.stepNumber)}
+              toggleStep={onToggleStep}
+              onTaskClick={props.onTaskClick}
+              key={step.stepNumber}
+            />
+          );
+        })}
+    </SectionAccordion>
+  );
+
+  const renderStartSection = (
+    <SectionAccordion
+      key={sectionNamesInRoadmap[1]}
+      sectionType={sectionNamesInRoadmap[1]}
+      mini={true}
+      isDividerDisabled={displayBusinessStructurePrompt}
+    >
+      {displayBusinessStructurePrompt ? (
+        <BusinessStructurePrompt isButtonHidden={true} />
```

Comment:
It seems like this is never being set with isButtonHidden = false?

Where is the logic that changes this prompt based on if we're on the task or not?